tests/1-test-union/src/app/Client.java

```diff
@@ -95,26 +93,26 @@ public static void main(String[] args) throws IOException, JNCException {
         Union.enable();
         client.init();
         NodeSet configs = client.getConfig();
-        
+
         // Get (first) config with name "c"
         C cConfig = getCConfig(configs);
-        
+
         // Clone a backup configuration for rollback purposes
         YangElement backup = cConfig.clone();
 
         String configAsXML = cConfig.toXMLString();
         System.out.println("Initial config:\n" + configAsXML);
         System.out.println("HashCode of the config XML string: " +
                 configAsXML.hashCode());
-        
+
 //        // Increment number of servers at each l by one
-        ElementChildrenIterator lIterator = cConfig.lIterator();
+//        ElementChildrenIterator lIterator = cConfig.lIterator();
```

Comment:
Remove if not needed; also, I think the surrounding commented-out code can be removed, even if the linter is ok with it.

Reply:
rolled back the change - left for later cleanup/code enhancement...